data.sql

```diff
@@ -13,3 +13,29 @@ VALUES ('Agumon', '2020-02-03', 0, true, 10.23),
        ('Blossom', '1998-10-13', 3, true, 17),
        ('Ditto', '2022-05-14', 4, true, 22)
 ;
+
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34),
+     ('Jennifer Orwell', 19),
+     ('Bob', 45),
+     ('Melody Pond', 77), 
+     ('Dean Winchester', 14),
+     ('Joddie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES('Pokemon'),
+      ('Digimon');
+
+UPDATE animals
+SET species_id = 2
+WHERE name LIKE '%mon';
+
+UPDATE animals 
+SET species_id = 1
+WHERE name NOT LIKE '%mon';
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
```

Comment:
- [ ] [OPTIONAL] Well done on this project. your code is well written. I just want to make a suggestion or give another perspective. You could dynamically update this table like 👇🏾 

```sql
   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Bob')
   WHERE name = 'Devimon' OR name = 'Plantmon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Melody Pond')
   WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
   WHERE name = 'Angemon'  OR name = 'Boarmon';
```

In most cases, you will not always know the particular Id but you will be provided with a name. So you use the **_name_** to find the **_id_**